src/async/Client.js

```diff
@@ -113,6 +115,14 @@ module.exports = class Client {
     return `${host}${path}`;
   }
 
+  getAlfredUrl() {
+    const host = process.env.ZENATON_ALFRED_URL
+      ? process.env.ZENATON_ALFRED_URL
+      : ZENATON_ALFRED_URL;
+
+    return `${host}`;
```

Comment:
?! 🤔like the consistency with other getters, but look like weird.

Reply:
I forgot to remove the template syntax thanks